Assets/LeapMotion/Core/Scripts/Utils/Utils.cs

```diff
@@ -720,6 +804,53 @@ public struct ChildrenEnumerator : IEnumerator<Transform> {
 
     #endregion
 
+    #region Runtime Gizmo Utils
+
+    /// <summary>
+    /// Draws a simple XYZ-cross position gizmo at the target position, whose size is
+    /// scaled relative to the main camera's distance to the target position (for reliable
+    /// visibility).
+    /// 
+    /// You can also provide a color argument and lerp coefficient towards that color from
+    /// the axes' default colors (red, green, blue). Colors are lerped in HSV space.
+    /// </summary>
+    public static void DrawPosition(this RuntimeGizmoDrawer drawer, Vector3 pos,
```

Comment:
Any reason you want this in the Utils instead of actually in the runtime gizmo drawer itself?